daemon/cluster/executor/container/executor.go

```diff
@@ -162,12 +173,17 @@ func (e *executor) Configure(ctx context.Context, node *api.Node) error {
 	}
 
 	_, err := e.backend.SetupIngress(clustertypes.NetworkCreateRequest{
-		ID: na.Network.ID,
+		ID: ingressNA.Network.ID,
 		NetworkCreateRequest: types.NetworkCreateRequest{
-			Name:          na.Network.Spec.Annotations.Name,
+			Name:          ingressNA.Network.Spec.Annotations.Name,
 			NetworkCreate: options,
 		},
-	}, na.Addresses[0])
+	}, ingressNA.Addresses[0])
+	if err != nil {
+		return err
+	}
+
+	err = e.backend.AddLBAttachments(lbAttachments)
```

Comment:
The feeling here is that if there is not ingress network there is no lb configured, is it correct or there is a different code path followed?

Reply:
If it is not ingress network, then we save the lb attachment (actually, only the per network IP address).
We only  use the lb IP address when we create a service.